CHANGELOG.md

```diff
@@ -6,12 +6,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 ### Changed
+* New CHANGELOG format (@thomasfedb)
+* `cache_id` is now less collision-prone thanks to a counter (@stillwaiting and @mtsmfm [#1866](https://github.com/carrierwaveuploader/carrierwave/pull/1866))
 
 ### Removed
 
 ### Fixed
+* Fix require RMagick deprecation warning (@thomasfedb and @bensie [#1788](https://github.com/carrierwaveuploader/carrierwave/pull/1788))
+* Fix dependency and config options in CI build (@mehlah [#1847](https://github.com/carrierwaveuploader/carrierwave/pull/1847))
```

Comment:
This can be removed as it happened prior to us branching `0.11-stable` from `0.10-stable`. I know that the same applies to the RMagick fix, however I'm deciding if that change will be removed from to `0.10-stable` branch.